docs/reference/transform/setup.asciidoc

```diff
@@ -5,39 +5,95 @@
 <titleabbrev>Setup</titleabbrev>
 ++++
 
-To use the {transforms}, you must have the
-{subscriptions}[appropriate license] and at least one
-<<transform-setup-nodes,{transform} node>> in your {es} cluster. If {stack}
-{security-features} are enabled, you must also ensure your users have the
-<<transform-privileges,necessary privileges>>.
-
 [discrete]
-[[transform-setup-nodes]]
-== {transform-cap} nodes
+[[requirements-overview]]
+== Requirements overview
+
+To use {transforms}, you must have:
 
-To use {transforms}, there must be at least one {transform} node in your cluster.
-If you want to control which nodes run {transforms}, add or remove `transform`
-from the `node.roles` setting on some nodes. For more information, see
-<<modules-node>> and <<transform-settings>>.
+[%interactive]
+- [ ] the {subscriptions}[appropriate license]
```

Comment:
These checkboxes are clickable in docs preview.
Is it intended? IMO the user could get a wrong impression that they are configuring something with these checkboxes...

Reply:
Yes, they're just an alternative to bullets. We've used them in https://www.elastic.co/guide/en/machine-learning/master/setup.html for many releases with no complaints so I think it's okay to leave them until we get into the new docs system and see what options are available there.

Reply:
I've gone ahead and removed them in 63140b77aa69eb69981e9d32b7f26eb0c738b419